test/modules/TestAnalyzer/TestResultReporter.cs

```diff
@@ -101,16 +101,54 @@ static ModuleMessagesReport GetReceivedMessagesReport(string moduleId, double to
                 }
             }
 
-            // check if last message is older
-            if (DateTime.Compare(lastMessageDateTime.AddMilliseconds(toleranceInMilliseconds), endDateTime) < 0)
+            bool areMessagesMissing = missingCounter > 0;
+            bool areLatestMessagesTooOld = DateTime.Compare(lastMessageDateTime.AddMilliseconds(toleranceInMilliseconds), endDateTime) < 0;
+
+            if (areMessagesMissing)
+            {
+                Logger.LogInformation($"Module {moduleId}: missing messages");
+            }
+
+            if (areLatestMessagesTooOld)
             {
                 Logger.LogInformation($"Module {moduleId}: last message datetime={lastMessageDateTime} and end datetime={endDateTime}");
-                return new ModuleMessagesReport(moduleId, StatusCode.OldMessages, totalMessagesCounter, $"Missing messages: {missingCounter}. No messages received for the past {toleranceInMilliseconds} milliseconds.", lastMessageDateTime, missedMessages, Settings.Current.TestInfo);
             }
 
-            return missingCounter > 0
-                ? new ModuleMessagesReport(moduleId, StatusCode.SkippedMessages, totalMessagesCounter, $"Missing messages: {missingCounter}.", lastMessageDateTime, missedMessages, Settings.Current.TestInfo)
-                : new ModuleMessagesReport(moduleId, StatusCode.AllMessages, totalMessagesCounter, "All messages received.", lastMessageDateTime, Settings.Current.TestInfo);
+            (StatusCode statusCode, string statusMessage) = GetStatus(areMessagesMissing, areLatestMessagesTooOld, missingCounter, toleranceInMilliseconds);
+
+            return new ModuleMessagesReport(moduleId, statusCode, totalMessagesCounter, statusMessage, lastMessageDateTime, missedMessages, Settings.Current.TestInfo);
+        }
+
+        static (StatusCode, string) GetStatus(bool areMessagesMissing, bool areLatestMessagesTooOld, long missingCounter, double toleranceInMilliseconds)
+        {
+            string missingMessagesStatus = $"Missing messages: {missingCounter}. ";
```

Comment:
nit: remove last space character at the end and add space in below usage, for example, line 133.